pkg/cmd/pulumi/about.go

```diff
@@ -423,7 +426,11 @@ func (current currentStackAbout) String() string {
 			Rows:    rows,
 		}.String() + "\n"
 	}
-	return fmt.Sprintf("Current Stack: %s\n\n%s\n%s", current.Name, resources, pending)
+	stackName := current.Name
+	if stackName != current.FullyQualifiedName {
+		stackName += fmt.Sprintf(" (fully qualified to %q)", current.FullyQualifiedName)
+	}
+	return fmt.Sprintf("Current Stack: %s\n\n%s\n%s", stackName, resources, pending)
```

Comment:
So this prints like:
```
Current Stack: stack (fully qualified to org/proj/stack)
```

I wonder if the "fully qualified to" text is needed. I wonder if the non-qualified name is even needed if we're doing a text print, like we clearly want to keep it as is for the json case but maybe this should just always print the fully qualified name now.

Reply:
Good point. I'll just print out the FQN if it exists.